examples/tensorflow/tf1/README.md

```diff
@@ -0,0 +1,55 @@
+## MLflow with TensorFlow 1.X
+
+In this example, we train a `tf.estimator.DNNRegressor` on the TensorFlow [Boston Housing dataset](https://www.cs.toronto.edu/~delve/data/boston/bostonDetail.html).
+With MLflow, we use `mlflow.tensorflow.autolog()` along with `mlflow.log_param()` and `mlflow.log_metric()` in order to capture data about how we trained the target model.
+We also use MLflow in order to generically save the model as a `mlflow.pyfunc`, then load it back to predict on pandas DataFrames.
+
+### Code related to MLflow:
+* [`mlflow.tensorflow.autolog()`](https://www.mlflow.org/docs/latest/tracking.html#automatic-logging-from-tensorflow-and-keras-experimental):
+This is an experimental api that logs ML model artifacts and TensorBoard metrics created by the `tf.estimator` we are using.
+The TensorBoard metrics are logged during training of the model. By default, MLflow autologs every 100 steps.
+The ML model artifact creation is handled during the call to `tf.estimator.export_saved_model()`.
+
+* [`mlflow.log_param()`](https://mlflow.org/docs/latest/python_api/mlflow.html#mlflow.log_param):
```

Comment:
Does autolog log any params on its own? If so, we could get rid of the `mlflow.log_param` from our examples, I think it gets confusing the more MLflow code that's present

Reply:
Yup, autolog logs `batch_size` and `train_steps` if run with `mlflow run` (not directly from the command line). We can take `log_param` out.